src/frontend/App/hooks/useHentVedtak.ts

```diff
@@ -19,7 +19,7 @@ export const harVedtaksresultatMedTilkjentYtelse = (
 };
 
 export const useHentVedtak = (
-    behandlingId: string
+    behandlingId: string | undefined
```

Comment:
@charliemidtlyng er dette en grei endring? Er for å kunne hente vedtak for forrigeBehandlingId

Reply:
Ja, du må bare null-sjekke behandlingId før du gjør en get-request til ef-sak, som du har gjort lengre nede